### PR TITLE
ET-3392: update demo card design

### PR DESCRIPTION
**Pull request by frankpepermans**
### What 🕵️ 🔍

- Update the discovery cards to match the newly proposed design

See [in figma](https://www.figma.com/file/hGAst3K9ZQIHmQoCNjbgaZ/Discovery-App-%7C-Design?node-id=3495%3A5419)

----------

### References 📝 🔗

- [ET-3392](https://xainag.atlassian.net/browse/ET-3392)

----------

### CODE REVIEW

File: application/lib/presentation/discovery_card/widget/discovery_card_static.dart [hunk 1]
```diff
@@ -186,7 +195,7 @@ class _DiscoveryCardStaticState
         left: R.dimen.unit3,
         right: R.dimen.unit3,
         bottom: R.dimen.readerModeBottomPadding,
-        top: size.height * _kImageFractionSize,
+        top: size.height / 2 + R.dimen.unit15,
```
Comment (carminetoriello) [inline application/lib/presentation/discovery_card/widget/discovery_card_static.dart:198]: ❓ Will adding always R.dimen.unit15 look good with every type of device screen size ?

File: application/lib/presentation/images/widget/arc.dart [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        );
+      },
+    );
+
+    return LayoutBuilder(
+        builder: (context, constraints) => Stack(
+              children: [
+                Positioned.fill(
+                  bottom:
+                      constraints.maxHeight / 2.5 * (1.0 - widget.fractionSize),
```
Comment (carminetoriello) [inline application/lib/presentation/images/widget/arc.dart:57]: ❓ Just out of curiosity, how the 2.5 value has been chosen ? :)

File: bin/main.dart [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  final S entryA;
+  final T entryB;
+
+  const CartesianProductResult(
+    this.i,
+    this.j, {
+    required this.entryA,
+    required this.entryB,
+  });
+}
```
Comment (carminetoriello) [inline bin/main.dart:80]: ❓ Is this file needed or it was just for testing ?

**Review by carminetoriello — COMMENTED** [review]
Tested and seems to work as intended!

Left some comments. Two are not blocking but please check the comment regarding the `bin/main.dart` file :)

File: bin/main.dart (hunk 3 above)
Comment (MichaelKMalak) [reply bin/main.dart:80]: oh true! removing it

File: application/lib/presentation/discovery_card/widget/discovery_card_static.dart (hunk 1 above)
Comment (MichaelKMalak) [reply application/lib/presentation/discovery_card/widget/discovery_card_static.dart]: [3cb2ce6](https://github.com/xaynetwork/xayn_discovery_app/pull/655/commits/3cb2ce629223ae1bbb9d9275f177941f1dea0c85)

**Review by carminetoriello — APPROVED** [review]
Looks good to me!! 🚀

**Comment by shushanAspaturyan** [comment]
testing passed
